From 2b4d27401fd3ce7622dfce978c0a18ba3c96b784 Mon Sep 17 00:00:00 2001
From: MphoKhotleng <54142532+MphoKhotleng@users.noreply.github.com>
Date: Wed, 8 Apr 2020 04:56:42 +0200
Subject: [PATCH 2/7] adding instructions to a text file

---
 part1_instructions | 19 +++++++++++++++++++
 1 file changed, 19 insertions(+)
 create mode 100644 part1_instructions

diff --git a/part1_instructions b/part1_instructions
new file mode 100644
index 0000000..84ffa2d
--- /dev/null
+++ b/part1_instructions
@@ -0,0 +1,19 @@
+Part 1: Creating a database
+
+1.Create a database called “Umuzi”.
+
+2.Create the following tables in the Umuzi database:
+
+Customers
+Employees
+Orders
+Payments
+Products
+
+3.Create a primary key for each table with auto-increment (make sure you correctly specify the data types, e.g. the ID field should be int).
+
+4.Create foreign keys so that every ID in the order table references an existing ID in the tables referenced (e.g., ProductID, EmployeeID, etc).
+
+5.INSERT the records in the tables below into the table you created in step 2.
+
+6.Document what information is stored in your database. Be sure to say what information is kept in what table, and which keys link the records between tables.
-- 
2.20.1

