From 069218667002bef75c076597ad12f8a4502abdac Mon Sep 17 00:00:00 2001
From: MphoKhotleng <54142532+MphoKhotleng@users.noreply.github.com>
Date: Wed, 8 Apr 2020 06:52:59 +0200
Subject: [PATCH 4/7] finishing off. no. 16 ISNULL

---
 part2_queries | 59 ++++++++++++++++++++++++++++++++++++++++-----------
 1 file changed, 47 insertions(+), 12 deletions(-)

diff --git a/part2_queries b/part2_queries
index dcb0197..219fd01 100644
--- a/part2_queries
+++ b/part2_queries
@@ -1,26 +1,61 @@
-1.SELECT * FROM Customers;
+1.SELECT * 
+FROM Customers;
 
-2.SELECT FirstName, LastName FROM Customers;
+2.SELECT FirstName, LastName 
+FROM Customers;
 
-3.SELECT FirstName, LastName FROM Customers
+3.SELECT FirstName, LastName 
+FROM Customers
 WHERE CustomerID = 1;
 
-4. UPDATE customers SET firstname = 'Lerato', lastname = 'Mabitso'
-   WHERE customerid = 1;
+4.UPDATE customers 
+SET firstname = 'Lerato', lastname = 'Mabitso'
+WHERE customerid = 1;
 
-5.DELETE FROM Customers WHERE CustomerID = 2;
+5.DELETE FROM Customers 
+WHERE CustomerID = 2;
 
-6.SELECT COUNT(DISTINCT(Status)) FROM Orders;
+6.SELECT COUNT(DISTINCT(Status)) 
+FROM Orders;
 
-7.SELECT MAX(Amount) FROM Payments;
+7.SELECT MAX(Amount) 
+FROM Payments;
 
-8.SELECT * FROM Customers
+8.SELECT * 
+FROM Customers
 ORDER BY Country;
 
-9.SELECT * FROM Products
-WHERE BuyPrice BETWEEN 100 AND 600;
+9.SELECT * 
+FROM Products
+WHERE buyprice BETWEEN 100 AND 600;
 
-10.SELECT * FROM Customers
+10.SELECT * 
+FROM Customers
 WHERE Country = 'Germany' AND City = 'Berlin';
 
+11.SELECT * 
+FROM Customers
+WHERE City = 'Cape Town' OR City = 'Durban';
+
+12.SELECT * 
+FROM Products
+WHERE buyprice > 500;
+
+13.SELECT SUM(Amount) 
+FROM Payments;
+
+14.SELECT COUNT(Status) 
+FROM Orders
+WHERE Status = 'Shipped';
+
+15.SELECT (buyprice/12) 
+FROM products;
+
+16.COMEBACK
+
+17.SELECT * 
+FROM Products
+WHERE Description LIKE 'Turnable front wheels%';
+
+
 
-- 
2.20.1

